Cargo.toml

```diff
@@ -15,6 +15,7 @@ include = [
     "/README.md",
```

Comment:
@sellout you should add yourself to "authors".

Reply:
A :tada: means that the comment has been addressed in https://github.com/ZcashFoundation/zcash_script/pull/197